### ISSUE TITLE
Share an alarm

### ISSUE THREAD

**Issue by dllud** [drpout/boilr#249]
Sometimes, although less often than #248, I feel the need to share an entire alarm. I would like to share it in a way that allows someone else read it and import into Boilr. Perhaps we could develop our own human-readable URI like `boilr:exchange,pair,up,low`.

**Comment by algazarra** [comment]
Interesting! However we will need to unclutter the alarm. let's say for ex. start using μ for long digit prices in order to include a vertical 3 dots button or something like it.

> human-readable URI like boilr:exchange,pair,up,low
+1

**Comment by dllud** [comment]
> we will need to unclutter the alarm [...] in order to include a vertical 3 dots button

That's true if we wanted to add the share button to the alarm list view. Though:
1. I think it is not necessary to add it there, and
2. I do like the current alarm layout. It will be difficult to do better. Any info we may hide is value info.

I was thinking about placing the share button on the action bar of the Alarm Settings activity. It makes some sense to have it there. That's the place where you have all the alarm properties, thus the place were you can share them.

**Comment by dllud** [comment]
Since, with #268, we are changing the alarm list display, there will probably be some free space to add the share button over there.
